rules/opentitan.bzl

```diff
@@ -426,9 +427,16 @@ def _elf_to_scrambled_rom_impl(ctx):
                 src.basename.replace("." + src.extension, ""),
             ),
         )
+        digest = ctx.actions.declare_file(
+            "{}.39.scr.vmem.digest.h".format(
+                # Remove ".elf" from file basename.
+                src.basename.replace("." + src.extension, ""),
```

Comment:
is it a digest of the VMEM file or a digest of the scrambled ROM? if the latter maybe: `"{}.39.scr.digest.h" ?

Reply:
Moreover, do we need to modify the bazel rules, or can you just use this target here: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/BUILD;drc=742df4f26d8ca6a4a5b0877dc0d6e72ab388bdcd;l=505

And add one for the test rom if you need it?

Reply:
I will double check, @luismarques did the hash part, I just helped with the bazel.

Reply:
I checked with @luismarques and this is different: this test checks that the ROM digest reported by ROM_CTRL matches what is computed by the scrambler. This is a cSHAKE hash. The code you pointed to uses SHA256 and the test is also different since the actually computes the hash instead of relying on the ROM_CTRL digest.
Another difference is this test is automated whereas the one you pointed to is a manual process for release I think?